OpenRA.Game/Map/MapGrid.cs

```diff
@@ -126,7 +126,7 @@ public class MapGrid : IGlobalModData
 
 		public CellRamp[] Ramps { get; }
 
-		internal readonly CVec[][] TilesByDistance;
+		public readonly CVec[][] TilesByDistance;
```

Comment:
This is only used within `DefaultMap`. Move it there instead of making it public.